npm/gen-root.ts

```diff
@@ -47,8 +47,13 @@ async function genServerPackage(buildDefinitions: string[]) {
         type: 'module',
         version: packageVersion,
         optionalDependencies,
+        scarfSettings: {
+            defaultOptIn: true
```

Comment:
```suggestion
            defaultOptIn: true,
            allowTopLevel: true
```

Reply:
@tusharmath i have made the changes please review it